public/data/planet/aoi.json

```diff
@@ -0,0 +1,51 @@
+{
+        "type": "Polygon",
```

Comment:
@sarveshpro is this a test file? why do we need this?

Reply:
This is the available area of Planet Labs data. I check if the project is inside it or fallback to sentinel